libs/elements/cards/src/lib/components/elewa-news-item-card/elewa-news-item-card.component.html

```diff
@@ -0,0 +1,14 @@
+<div *ngFor="let item of newsItem" class="news-item">
+  <!-- title -->
+  <div class="title">
+    {{ item.title }}
```

Comment:
This component should only render one item. No need for the loop

Reply:
i have fixed [<div class="news-item">
  <!-- title -->
  <div class="title">
    {{ newsItem[0].title }}
  </div>
  <!-- description -->
  <div class="description">
    {{ newsItem[0].description }}
  </div>
  <!-- button text -->
  <div class="button-text">
    {{ newsItem[0].buttonText }}
  </div>
</div>
]